ZenPacks/zenoss/HttpMonitor/datasources/HttpMonitorDataSource.py

```diff
@@ -133,16 +133,13 @@ def collect(self, config):
         regex = ds0.params['regex']
         caseSensitive = ds0.params['caseSensitive']
         invert = ds0.params['invert']
+        onRedirect = ds0.params['onRedirect']
+
+        onRedirect = str(onRedirect)
+
+        if onRedirect in ('False', ''):
+            onRedirect = "fail"
```

Comment:
What is the value of `onRedirect` if it's 'True'?  Shouldn't it be set to "follow"?

Reply:
I've added `onRedirect` == True condition